.github/workflows/ci.yml

```diff
@@ -0,0 +1,230 @@
+        if: matrix.config.cxx == 'clang++-3.9'
+        shell: bash
+        run: ${{ matrix.config.install_command }} ${{ matrix.config.package }}
+
+      - name: Install Clang 7 on Linux
+        if: startsWith(matrix.config.os, 'ubuntu') && matrix.config.cxx == 'clang++-7'
+        shell: bash
+        run: ${{ matrix.config.install_command }} ${{ matrix.config.package }}
+
+      - name: Install Clang 7 on macOS
+        if: startsWith(matrix.config.os, 'macos') && matrix.config.cxx == 'clang++-7'
+        shell: bash
+        run: |
+          ${{ matrix.config.install_command }} ${{ matrix.config.package }}
+          ln -s /usr/local/opt/llvm@7/bin/clang++ /usr/local/opt/llvm@7/bin/clang++-7
+          echo "::add-path::/usr/local/opt/llvm@7/bin" # Only available in subsequent steps!
+
+      - name: Install Libraries on Linux
+        if: startsWith(matrix.config.os, 'ubuntu')
+        shell: bash
+        run: ${{ matrix.config.install_command }} libboost-dev libbz2-dev libxml2-utils zlib1g-dev
+
+      - name: Install Libraries on macOS
+        if: startsWith(matrix.config.os, 'macos')
+        shell: bash
+        run: ${{ matrix.config.install_command }} boost
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+
+      - name: Install Python dependencies
+        shell: bash
+        run: |
+          pip install setuptools wheel
+          pip install -r seqan/manual/requirements.txt
+
+      - name: Prepare ccache
+        id: ccache_prepare
+        shell: cmake -P {0}
+        run: |
+          string(TIMESTAMP current_date "%Y-%m-%d-%H;%M;%S")
+          message("::set-output name=timestamp::${current_date}")
+
+      - name: Load ccache
+        uses: actions/cache@v1.1.0
+        with:
+          path: .ccache
+          key: ${{ matrix.config.name }}-ccache-${{ steps.ccache_prepare.outputs.timestamp }}
+          restore-keys: |
+            ${{ matrix.config.name }}-ccache-
+
+      # Also pass python path: https://github.com/actions/setup-python/issues/121
+      - name: Configure tests
+        env:
+          CXX: ${{ matrix.config.cxx }}
+          CC: ${{ matrix.config.cc }}
+          BUILD_TYPE: ${{ matrix.config.build_type }}
+          CXXFLAGS: ${{ matrix.config.cxxflags }}
+        shell: bash
+        run: |
+          mkdir seqan-build
+          cd seqan-build
+          cmake ../seqan -DCMAKE_BUILD_TYPE=${BUILD_TYPE} -DSEQAN_TRAVIS_BUILD:BOOL=ON -DCMAKE_CXX_FLAGS="-DSEQAN_VERSION_CHECK_OPT_IN=YES ${CXXFLAGS}" -DPython3_ROOT_DIR=${pythonLocation}
+
+      - name: Build and run tests
+        env:
+          CCACHE_BASEDIR: ${{ github.workspace }}
+          CCACHE_DIR: ${{ github.workspace }}/.ccache
+          CCACHE_COMPRESS: true
+          CCACHE_COMPRESSLEVEL: 6
+          CCACHE_MAXSIZE: 5G
+        shell: bash
+        run: |
+          cd seqan-build
+          cmake --build . --parallel 2
+          ctest . -j2 --output-on-failure
```

Comment:
Can we split the test part so that we can see how long it takes to build / test.

Reply:
Yes, this needs to be updated to the seqan3 version anyways.

And probably even use Cmake scripting (see link in description) to properly work on windows.

And we need to figure out how to use the correct msvc version - it is installed but I couldn't get it to use the correct version.

Reply:
In the past I used this command:

```
cmake -G"Visual Studio 14 2015 Win64" ../src
```